docs/src/configuration.md

```diff
@@ -116,6 +75,45 @@ For more complex configurations you may find it simpler to use
 See [base16.nix](https://github.com/SenchoPens/base16.nix) documentation for
 usage examples.
 
+## Wallpaper
+
+To set a wallpaper, provide a path or an arbitrary derivation:
+
+- ```nix
+  {
+    stylix.image = ./wallpaper.png;
+  }
+  ```
+
+- ```nix
+  {
+    stylix.image = pkgs.fetchurl {
+      url = "https://www.pixelstalk.net/wp-content/uploads/2016/05/Epic-Anime-Awesome-Wallpapers.jpg";
+      sha256 = "enQo3wqhgf0FEPHj2coOCvo7DuZv+x5rL/WIo4qPI50=";
+    };
+  }
+  ```
+
+If `stylix.base16Scheme` is undeclared, Stylix generates the color scheme with a
+[genetic algorithm](https://en.wikipedia.org/wiki/Genetic_algorithm). Note that
```

Comment:
Perhaps make it extra clear that the generated scheme is based on the wallpaper:

```suggestion
If `stylix.base16Scheme` is undeclared, Stylix generates a color scheme based on the wallpaper using a
[genetic algorithm](https://en.wikipedia.org/wiki/Genetic_algorithm). Note that
```